src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -93,14 +99,14 @@ private UUID saveNewLetter(ILetterRequest letter, String messageId, String servi
         );
 
         // TODO: encrypt zip content
-
         Letter dbLetter = new Letter(
             id,
             messageId,
             serviceName,
             mapper.valueToTree(letter.getAdditionalData()),
             letter.getType(),
-            zipContent
+            zipContent,
+            isEncryptionEnabled
```

Comment:
this should always be `false` (for now). feature toggle doesn't do anything

Reply:
If we use value from feature toggle we could then enable/disable it on specific env say sandbox and test it.

Reply:
but file is not actually being encrypted

Reply:
Feel free to log something instead (when file is supposed to be encrypted)